config/default/kustomization.yaml

```diff
@@ -20,41 +18,7 @@ bases:
 patchesStrategicMerge:
 - manager_image_patch.yaml
 - manager_label_patch.yaml
-
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
```

Comment:
Should we leave these here commented out for when we need to enable webhooks?

Reply:
Yes yes yes!

Reply:
ok